Dockerfile

```diff
@@ -1,14 +1,19 @@
 FROM nginx:1.10
 MAINTAINER Wyatt Pearsall<Wyatt.Pearsall@cfpb.gov>
+ARG SKIP_JS_BUILD
 
-RUN mkdir -p /usr/src/app
+RUN if [ -z ${SKIP_JS_BUILD+x} ]; then echo "Installing JS deps" && apt-get update && \
+    apt-get install -y curl && \
+    curl -sL https://deb.nodesource.com/setup_7.x | bash - && \
+    apt-get install -y nodejs; fi && mkdir -p /usr/src/app
 
 WORKDIR /usr/src/app
-# COPY . /usr/src/app
+
+COPY . /usr/src/app
 COPY nginx/nginx.conf /etc/nginx/nginx.conf
 
-# RUN chown -R root /usr/src/app && chmod u+rwx /usr/src/app
+RUN if [ -z ${SKIP_JS_BUILD+x} ]; then echo "Building JS" && npm install; fi
```

Comment:
Whenever I build this without `SKIP_JS_BUILD` set, and I run it without the dev volume mounted, I end up with a near-empty `dist` directory.  To fix this, I had to add `&& npm run build` to this line.  Is see there is a [`postinstall`](https://github.com/cfpb/hmda-platform-ui/pull/455/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R19) script, that seems to do this, and I assume run _post-`install`_, but I don't think its workin'...if that's the intent.